# Name of the agent storing the octomap
string agent_name

# Entropy value
std_msgs/Float32 value
